### DISCUSSION TITLE
Jetton tokens

### DISCUSSION THREAD

**Discussion by The247Nuggles** [community/community#144141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Please can anyone help me how to widraw my tokens please

**Comment by samus-aran** [comment]
Hi @The247Nuggles,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by The247Nuggles** [reply]
Okay thanx alot
